### PR TITLE
Fix walking contractions

### PR DESCRIPTION
**Pull request by caffodian**
I broke my local tests by upgrading Phantomjs by accident so I'll let Travis do it.

### CODE REVIEW

**Comment by caffodian** [comment]
Quick fixes for #30 .  will need to do more integration testing (seems to work when playing around, but should load a good sample of real docs.)

File: tests/typowalking/wordwalker.js [hunk 1]
```diff
@@ -293,7 +293,7 @@ bender.test( {
 			wordsReturned;
 
 		bot.setHtmlWithSelection(
-			'<p>paragraph<br/>break<sup>superscript</sup> paragraph<sub>subscript</sub>'
+			'<p>paragraph<br/>break<sup>superscript</sup> paragraph<sub>subscript</sub></p>'
```
Comment (winhamwr) [inline tests/typowalking/wordwalker.js:296]: Was the missing closing paragraph unintentional, previously?

**Comment by winhamwr** [comment]
Looks good!

File: tests/typowalking/wordwalker.js (hunk 1 above)
Comment (caffodian) [reply tests/typowalking/wordwalker.js:296]: pretty sure it was unintentional given the test title.  it does fix itself though

**Pull request merged**